tool/win32gen/data/win32_functions.json

```diff
@@ -3747,12 +3747,12 @@
     "NetFreeAadJoinInformation": {
         "prototype": "VOID NET_API_FUNCTION NetFreeAadJoinInformation(\n  [in, optional] PDSREG_JOIN_INFO pJoinInfo\n);",
         "comment": "Frees the memory allocated for the specified DSREG_JOIN_INFO structure, which contains join information for a tenant and which you retrieved by calling the NetGetAadJoinInformation function.",
-        "minimumWindowsVersion": "WIN10_RS1"
```

Comment:
In the Win32 metatadata, the minimum Windows version is specified as `windows10.0.10240` which translates to `WIN10_TH1`.